HHS/OPRE-OPS
# Ops 675/task 1060 form validation

## What changed

Adds form validation with the Vest library.  Adds POC with the Project Create form.

## Issue

#675 and #1060

## How to test

Goto Create a new Project and test to see if all form fields must be entered and select must be selected. Submit button will be disabled until all form validation is complete.

## Screenshots

<img width="1031" alt="image" src="https://github.com/HHS/OPRE-OPS/assets/4629398/25b5f86b-b711-4739-8f9e-2bf68cddd326">

## Links

[Vest docs](https://vestjs.dev/)
